app/views/users/show.haml

```diff
@@ -7,12 +7,12 @@
           .user-moiety
             = color_block(@user)
             = @user.moiety_name
-  %tbody
```

Comment:
I don't see the avatar as part of the header, really, though I grant it's not part of the regular body either. It's in a `%td` at the moment, though, so I'd err on putting it in the `%tbody`.

Reply:
(put it in a seperate `%tbody`)